### DISCUSSION TITLE
Implied relationships documentation is confusing

### DISCUSSION THREAD

**Discussion by beffge** [structurizr/structurizr.github.io#10] [General]
Hi Simon,
thanks again for all your effort put into the C4 Model & Structurizr.

I was just now reading about the implied relationships documentation [here](https://docs.structurizr.com/dsl/cookbook/implied-relationships/) and followed the link to [here](https://docs.structurizr.com/java/implied-relationships#createimpliedrelationshipsunlessanyrelationshipexistsstrategy) as well. Maybe it's just me, but I found a bit it confusing.

In the first link you write

> By default, the Structurizr DSL will create “implied relationships” automatically

and that this behaviour can be disabled via `!impliedRelationships` set to `false`. But when following the second link, you state `By default, the Structurizr for Java library will not create implied relationships.`.

I guess since dsl handles it different than the java library, maybe it would be good to make it more explicit i.e.
`By default, the Structurizr for Java library will not create implied relationships. In contrary to the DSL which uses CreateImpliedRelationshipsUnlessAnyRelationshipExistsStrategy as default.`

Just my 2 cents.

Additional to this, [here](https://docs.structurizr.com/java/implied-relationships#createimpliedrelationshipsunlesssamerelationshipexistsstrategy) you are talking about `CreateImpliedRelationshipsUnlessSameRelationshipExistsStrategy` but in the example code `CreateImpliedRelationshipsUnlessAnyRelationshipExistsStrategy` is used. Is this intended?

Thanks and have a nice day

**Comment by simonbrowndotje** [comment]
Thanks for the feedback - I've made some tweaks.

**Comment by beffge** [comment]
Nice. Should hopefully help to reduce confusion in the future. Thanks!
